examples/five_pl_logistic/five_pl_fit.ipynb

```diff
@@ -0,0 +1,934 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "execution_count": 42
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "While somewhat subtle, close observation reveals superior performance of the 5PL model for higher concentration samples. You should expect to see similar performance when your data has high levels of asymmetry.\n",
```

Comment:
Probably worth putting in one more cell explaining the superior performance, similar to here: https://www.brendan.com/5pl-curve-fitting/ section "Comparing 5PL and 4PL Curve Fits" where he plots the residuals.